README.md

```diff
@@ -44,6 +44,7 @@ Options
    --certFile value              TLS Certificate
    --keyFile value               TLS Certificate Key
    --logPrefix value             Setup custom log prefix
+   --disableNotifications        disable desktop notifications
```

Comment:
@resurtm can we actually make desktop notifications disabled by default (and add a flag to enable them)?